hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmInfo.java

```diff
@@ -30,6 +30,7 @@ public final class ScmInfo {
   private final String clusterId;
   private final String scmId;
   private final List<String> peerRoles;
+  private final Boolean scmRatisEnabled;
```

Comment:
Is this intentionally `Boolean` instead of `boolean`?  Since its value is set in the constructor from a `boolean` parameter, it doesn't seem to be ever set to `null`.

Reply:
Thank you very much for helping review the code! I will improve this part of the code.